.dumi/scripts/mirror-notify.js

```
@@ -0,0 +1,226 @@
+
+    notify.addEventListener('mouseleave', () => {
+      startProgressTimer();
+    });
+  }
+
+  // 超过 5 秒则判定网络不畅
+  const delayDuration = 5;
+
+  const reactTimeoutId = setTimeout(() => {
+    if (typeof (window[SIGN] || {}).YES === 'undefined') {
+      console.error(`antd.mirror-notify: 页面加载超过 ${delayDuration} 秒，可能是网络不畅。\n请尝试访问国内镜像站点。%c${officialChinaMirror}`, `color: ${primaryColor}; font-weight: bold;`);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)